<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>130</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pressure Plot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="show_plot_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Show Plot</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::NoButton</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="main_menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="about_action"/>
    <addaction name="setup_action"/>
   </widget>
   <addaction name="main_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="about_action">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="setup_action">
   <property name="text">
    <string>Setup</string>
   </property>
  </action>
  <action name="close_action">
   <property name="text">
    <string>Close Application</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
